apps/condo/domains/miniapp/access/AppMessageSetting.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Generated by `createschema miniapp.AppMessageSettings 'app?:Relationship:B2CApp:CASCADE; blockReason:Text; blockedNotificationTypes:Text; throttlingTime:Integer;'`
+ */
+
+const uniq = require('lodash/uniq')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { find } = require('@open-condo/keystone/schema')
+
+const { STAFF } = require('@condo/domains/user/constants/common')
+
+
+async function canReadAppMessageSetting ({ authentication: { item: user }, context }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return {}
+
+    if (user.type === STAFF) {
+        const userEmployees = await find('OrganizationEmployee', {
```

Comment:
Can we reuse some logic from here?
https://github.com/open-condo-software/condo/blob/f154008f7a906e7010847ec81b0bdd3d8c90f03d/apps/condo/domains/organization/utils/accessSchema.js

Since this utils was designed to:
1. Cache employees to avoid extra pg lookups
2. Avoid dealing with `OrganizationEmployee.where` in accesses, since it was one of the most common things where mistakes are made, so this filter now is specified only in 1 place

Reply:
- [ ] Add role `id` to cache https://github.com/open-condo-software/condo/blob/f154008f7a906e7010847ec81b0bdd3d8c90f03d/apps/condo/domains/organization/utils/accessSchema.js#L144-L147 Add function that returns role ids by user, and use it here